archive/governance/special-interest-groups/sig-web/README.md

```diff
@@ -1,17 +1,17 @@
 # Web Special Interest Group(Web SIG)
 
-Web SIG focus on website development around tidb, includding:
+Web SIG focus on website development around tidb, including:
```

Comment:
Can we delete sig-web all related about sig-web ? (even though they are archived)

Reply:
Why?

Reply:
It should be clean up years... ago as repositories in it are not `community oriented`.

Reply:
Make sense. Updating...

Reply:
Updated.